src/pages/app-bar/collapsible-app-bar/index.tsx

```diff
@@ -11,6 +11,12 @@ import { InfoListItem } from '@pxblue/react-components';
 const MAX_APP_BAR_HEIGHT = 128; // Specified by Material Design
 
 const useStyles = makeStyles((theme: Theme) => ({
+    appbarRoot: {
+        padding: 0,
```

Comment:
Is this actually doing anything?

Reply:
Yeah this is for the normal toolbar, not the collapsible one.